content/en/real_user_monitoring/guide/best-practices-tracing-native-ios-android-apps.md

```diff
@@ -0,0 +1,167 @@
+|---------------------------------------------|------------------------------------------------------------------|
+| `AndroidTracer.Builder.setSampleRate`                          | Controls the percentage of manually instrumented spans collected by the tracer.|
+| `DatadogInterceptor.Builder.setTraceSampler` | Controls the percentage of network requests traced for first-party hosts. |
+
+**Note**: `RUM.sessionSampleRate` controls RUM session sampling and does not affect trace sampling rates. When using the Trace SDK independently of RUM, trace sampling is controlled only by the parameters listed above.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### How sampling works
+
+Sampling affects different types of spans you create in your mobile app:
+
+- **Local span sampling** applies to manually instrumented spans (like performance profiling spans). It's controlled by the `Trace.sampleRate` parameter. For example, if you set this rate to 50, the Trace SD produces spans in 50% of the cases, and send all of those to Datadog. Visibility of those spans in the UI depends on your APM retention filters. Each span event includes the `_dd.agent_psr` field (the sampling rate) and `metrics._sampling_priority_v1` (1 for sampled, 0 for not sampled).
+
+- **Distributed trace sampling** applies to traces that cross service boundaries, such as network requests to your backend (relevant for the "Wrap a frontend-to-backend distributed trace" use case). This is controlled by the `urlSessionTracking.firstPartyHostsTracing.sampleRate` parameter for iOS and `DatadogInterceptor.Builder.setTraceSampler` parameter for Android. If set to 50, only half of backend requests have the sampled flag set to true, as indicated by the [W3C trace context][7]. All Datadog agents honor this decision, so you see 50% of distributed traces in the UI.
+
+Sampling rates are applied independently. The most restrictive rate determines what data is visible in the UI for a given session or trace. For example:
+- If you set a low RUM session sample rate (for example, 1%), only 1% of user sessions are recorded for RUM, but you can still trace all network requests within those sessions by setting the network tracing sample rate to 100%.
+- If you set a high trace sample rate but a low RUM sample rate, you may see traces without corresponding RUM data.
+
+**Example scenario:**  
+To sample 1% of all app sessions and trace all API network requests within those sessions:
+- Set `RUM.sessionSampleRate = 1` (controls RUM session sampling only)
+- Set `urlSessionTracking.firstPartyHostsTracing.sampleRate = 100` (controls trace sampling for network requests)
+
+Sampling decisions are communicated through trace headers, ensuring all services in a distributed trace use the same sampling choice.
+
+**Note**: The sampling behavior described above applies to iOS SDK `v2.9.0+` and Android SDK `v1.18.0+`. For earlier versions, see the respective SDK documentation for platform-specific sampling behavior.
```

Comment:
It might be worth putting this at the beginning of the sampling section? I think I might find it frustrating if I'm using an older version, read this topic, and then only realize at the end that it wasn't relevant to my version.